build.sbt

```diff
@@ -1,11 +1,9 @@
-import sbt.internal.ProjectMatrix
 lazy val V = _root_.scalafix.sbt.BuildInfo
 
 ThisBuild / tlBaseVersion := "0.0"
 
-// Cannot support Scala 3.x until Scalafix supports 3.x for semantic rules
-// lazy val scala3Version = "3.1.2"
-lazy val rulesCrossVersions = Seq(V.scala213, V.scala212)
+ThisBuild / crossScalaVersions := Seq(V.scala213, V.scala212)
```

Comment:
We'll find out eventually I guess, but we may need to train steward to re-generate the workflow on scalafix updates since I guess that will change the `crossScalaVersions`?

Reply:
At the very least, we can configure it for this repo.
https://github.com/scala-steward-org/scala-steward/blob/main/docs/repo-specific-configuration.md